index.html

```diff
@@ -2708,6 +2819,90 @@ <h3>
         </ol>
       </section>
     </section>
+    <section>
+      <h2>
+        <dfn>ProtocolHandlerItem</dfn> and its members
+      </h2>
+      <pre class="idl">
+        dictionary ProtocolHandlerItem {
+          required DOMString protocol;
+          required USVString url;
+        };
+      </pre>
+      <p>
+        Each <a>ProtocolHandlerItem</a> represents a protocol that the web
+        application wants to handle and the corresponding URL that should
+        handle the request. A user agent SHOULD use these values to register
+        the web application as a handler with the operating system. When the
+        user activates a protocol handler URL, the user agent SHOULD run
+        <a>Handling a protocol launch</a>.
+      </p>
+      <p class="note">
+        [[HTML]]'s <code>registerProtocolHandler</code> method allows web sites
+        to register themselves as possible handlers for particular protocols.
+        What constitutes valid <code>protocol</code> and <code>url</code>
+        values for <a>ProtocolHandlerItem</a>s is defined in that API. Also
+        note that the [[HTML]] API uses <code>scheme</code> where we use
+        <code>protocol</code> but the same restrictions apply.
+      </p>
+      <section data-dfn-for="ProtocolHandlerItem" data-link-for=
+      "ProtocolHandlerItem">
+        <h3>
+          <code>protocol</code> member
+        </h3>
+        <p>
+          The <dfn>protocol</dfn> member of a <a>ProtocolHandlerItem</a> is a
+          <a>string</a> that represents the protocol to be handled, such as
+          "mailto" or "web+auth".
+        </p>
+        <p>
+          The <a>protocol</a> member of a <a>ProtocolHandlerItem</a> is
+          equivalent to <code>registerProtocolHandler</code>'s
+          <code>scheme</code> argument defined in [[HTML]], and is processed in
+          the same manner.
+        </p>
+      </section>
+      <section data-dfn-for="ProtocolHandlerItem" data-link-for=
+      "ProtocolHandlerItem">
+        <h3>
+          <code>url</code> member
+        </h3>
+        <p>
+          The <dfn>url</dfn> member of a <a>ProtocolHandlerItem</a> is the
+          <a>URL</a> <a data-lt="within scope of a manifest">within the
```

Comment:
Please use "manifest scope" here, otherwise it might get confusing as "application scope" is not a defined thing.

Reply:
updated to "manifest scope".